src/transformers/configuration_utils.py

```diff
@@ -66,6 +66,9 @@ class PretrainedConfig(object):
             Whether cross-attention layers should be added to the model. Note, this option is only relevant for models
             that can be used as decoder models within the `:class:~transformers.EncoderDecoderModel` class, which
             consists of all models in ``AUTO_MODELS_FOR_CAUSAL_LM``.
+        tie_encoder_decoder_word_embeds (:obj:`bool`, `optional`, defaults to :obj:`True`)
```

Comment:
The config is only relevant to `EncoderDecoderConfig` -> can we move it there? 
This is different to `tie_encoder_decoder` because `tie_encoder_decoder` is also relevant for `T5`, `ProphetNet`.

Reply:
Im not familiar with `ProphetNet`, but I see that `T5` has shared Embedding matrix exclusively. I will move this parameter, assuming there will be no need for it in other future model types.